pkgs/by-name/pi/pixelorama/package.nix

```diff
@@ -1,64 +1,73 @@
-{ lib
-, stdenv
-, alsa-lib
-, autoPatchelfHook
-, fetchFromGitHub
-, godot3-headless
-, godot3-export-templates
-, libGLU
-, libpulseaudio
-, libX11
-, libXcursor
-, libXi
-, libXinerama
-, libXrandr
-, libXrender
-, nix-update-script
-, udev
+{
```

Comment:
I don't understand why `nix-update-script, udev` is separate.
I would favor alphabetical order here. While there isn't consensus on this, alphabetical order is objective.